apps/condo/domains/organization/components/OrganizationSelect.tsx

```diff
@@ -1,190 +1,162 @@
-    border: 1px solid transparent;
-    box-shadow: none;
-  }
-  &.ant-select-focused:not(.ant-select-disabled).ant-select:not(.ant-select-customize-input) .ant-select-selection-item,
-  &.ant-select:not(.ant-select-disabled):hover .ant-select-selection-item,
-  &.ant-select.ant-select-single.ant-select-open .ant-select-selection-item {
-    color: ${colors.white};
-  }
-  &.ant-select:not(.ant-select-disabled):hover .ant-select-arrow,
-  &.ant-select.ant-select-single.ant-select-open .ant-select-arrow {
-    color: ${colors.white};
-  }
-
-  &.ant-select:not(.ant-select-disabled):active .ant-select-selector {
-    background: ${gradients.sberActionInversed};
-    border: 1px solid transparent;
-  }
-
-  &.ant-select.ant-select-single.ant-select-open .ant-select-selector {
-    background: ${gradients.sberActionGradient};
-    border: 1px solid transparent;
-  }
-`
-
-// TODO(zuch): can't use emotion css here
-const optionStyle: React.CSSProperties = {
-    fontSize: '14px',
-    lineHeight: '20px',
-    backgroundColor: colors.white,
+function compareEmployees (lhs: OrganizationEmployeeType, rhs: OrganizationEmployeeType) {
+    lhs.organization.name
+    return get(lhs, ['organization', 'name'], '')
+        .toLowerCase()
+        .localeCompare(
+            get(rhs, ['organization', 'name'], '').toLowerCase()
+        )
 }
 
-const ORGANIZATION_SELECT_SHOW_ACTIONS: SelectProps<string>['showAction'] = ['focus', 'click']
+const DROPDOWN_OVERLAY_STYLES: CSSProperties = { maxWidth: 300, width: '100%' }
 
-export const OrganizationSelect: React.FC = () => {
+export const InlineOrganizationSelect: React.FC = () => {
     const intl = useIntl()
-    const EmptyMessage = intl.formatMessage({ id: 'Select' })
     const AddOrganizationTitle = intl.formatMessage({ id: 'pages.organizations.CreateOrganizationButtonLabel' })
 
+    const { breakpoints } = useLayoutContext()
+    const textSize: TypographyTextProps['size'] = !breakpoints.TABLET_LARGE ? 'small' : 'medium'
+
     const router = useRouter()
+
     const { user } = useAuth()
-    const selectRef = useRef(null)
     const { link, selectLink, isLoading: organizationLoading } = useOrganization()
-
     const userId = get(user, 'id', null)
-    const userIdFromSelectedLink = get(link, 'user.id', null)
 
-    const { objs: userOrganizations, allDataLoaded: organizationLinksLoaded } = OrganizationEmployee.useAllObjects(
-        { where: { user: { id: userId }, isRejected: false, isBlocked: false } }
-    )
+    const { objs: userEmployees, allDataLoaded: employeesLoaded } = OrganizationEmployee.useAllObjects({
+        where: {
+            user: { id: userId },
+            isRejected: false,
+            isBlocked: false,
+            isAccepted: true,
+        },
+    })
 
-    const organizationLinksLoading = !organizationLinksLoaded
+    // Note: Filter case where organization was deleted
+    const filteredEmployees = userEmployees.filter(employee => employee.organization)
```

Comment:
can we filter it in query above?

Reply:
Nope, since we're not using `deletedAt` filters in queries. So we need to check it manually in response.